docs/README.es-ES.md

```diff
@@ -111,6 +111,7 @@ $ yarn add beautiful-react-hooks
 * [useSwipe](useSwipe.md), [useHorizontalSwipe](useHorizontalSwipe.md) and [useVerticalSwipe](useVerticalSwipe.md)
 * [useSwipeEvents](useSwipeEvents.md)
 * [useConditionalTimeout](useConditionalTimeout.md)
+* [useCookie](docs/useCookie.md)
```

Comment:
Pay attention, this link will be broken cause you're already in the `/docs` folder. 
The correct link in this case will be: 

[useCookie](useCookie.md)

Reply:
Wooo nice. Thanks for that :)